x/epochs/client/cli/query.go

```diff
@@ -138,7 +139,7 @@ func GetCmdSecondsRemaining() *cobra.Command {
 			}
 
 			// duration: seconds
-			duration := int64(res.Epoch.Duration.Seconds())
+			duration := cast.ToInt64(res.Epoch.Duration.Seconds())
```

Comment:
These casts are great. Should we also be replacing instances of `strconv.FormatInt` with casts, e.g. line 150?

Reply:
This just returns strings, which I think can be arbitrarily long (no overflow risk) so I think using `strconv.FormatInt` is safe